libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -494,7 +493,6 @@ - (void)createAndStoreLogoutEvent:(NSError *)error user:(SFUserAccount*)user {
     NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
     attributes[@"errorCode"] = [NSNumber numberWithInteger:error.code];
     attributes[@"errorDescription"] = error.localizedDescription;
-    [SFSDKEventBuilderHelper createAndStoreEvent:@"userLogout" userAccount:user className:NSStringFromClass([self class]) attributes:attributes];
```

Comment:
I didn't notice the second line before 👍 . Looks like the containing method isn't needed either?